store/tree.go

```diff
@@ -153,14 +153,18 @@ func (ts *TreeStore) Remove(id string, s *Store) error {
 
 	key, err := ts.GetImageHash(id)
 	if err != nil {
-		return err
+		fmt.Fprintf(os.Stderr, "warning: tree store in a bad state, forcing removal\n")
```

Comment:
Logging is becoming a PITA I think. Anyway, prepend it with some prefix like "store: ".